tests/test_core_model.py

```diff
@@ -30,48 +42,43 @@ def test_info_simple_model(self, simple_supply):
         simple_supply.info()
 
 
-class TestAddMath:
-    @pytest.fixture
-    def storage_inter_cluster(self):
-        return build_model(
-            {"config.init.add_math": ["storage_inter_cluster"]},
-            "simple_supply,two_hours,investment_costs",
+class TestInitMath:
```

Comment:
Math tests have been subdivided into 3 groups for clarity:
- Testing the math init function directly.
- Testing the math update function directly.
- Testing overrides from Model() to the math.